weaviate/gql/get.py

```diff
@@ -1140,8 +1154,11 @@ def do(self) -> dict:
         """
         grpc_enabled = (  # only implemented for some scenarios
             self._connection.grpc_stub is not None
-            and self._near_ask is not None
-            and (isinstance(self._near_ask, NearVector) or isinstance(self._near_ask, NearObject))
+            and (
+                self._near_ask is None
```

Comment:
It used to be `is not None`, why is it `is None` now?

Reply:
before it required self._near_ask to be present. The new version makes self._near_ask optional, but if it is present it must be either NearObject or NearVector